### ISSUE TITLE
If single source is mentioned, don't show the dialog with Gallery and Camera

### ISSUE THREAD

**Issue by Thamizhan** [zendesk/belvedere#10]
There are two kind of source, Gallery and Camer, if we enable both int he Builder it is good to show the dialog to select Gallery or Camera, if we build it with either option, Still we are facing the dialog with single option

It is better not to show the dialog if we build with Single source and directly open the Camera or the Gallery based on the source mentioned.
![screenshot_20160718-142246](https://cloud.githubusercontent.com/assets/3370284/16909924/1f5a2bd8-4cf3-11e6-8e14-aaa9c937a28a.png)

**Comment by schlan** [comment]
Hey @Thamizhan, thank you for the suggestion, I think it's a valid FR. 
I'll create a PR https://github.com/zendesk/belvedere/pull/11. I let you know as soon as it's merged.

**Comment by dr03lf** [comment]
Hi @Thamizhan, #11 is merged. You can try it by using the latest `SNAPSHOT` version.

```
compile group: 'com.zendesk', name: 'belvedere', version: '1.2.0.1-SNAPSHOT'
```

**Comment by schlan** [comment]
Hi @Thamizhan, #11 is merged. You can try it by using the latest `SNAPSHOT` version.
a
```
compile group: 'com.zendesk', name: 'belvedere', version: '1.2.0.1-SNAPSHOT'
```

**Comment by schlan** [comment]
Fixed in `v1.2.0.1`
